### DISCUSSION TITLE
Dropdown for change branch order

### DISCUSSION THREAD

**Discussion by m0v3r** [gitextensions/gitextensions#9670] [Q&A] [answered]
### Environment

- GitExtensions version:
- OS version: 

### Issue description

Dropdown for change branch order after the latest updates is by name. It should be sorted by last used branch. Please fix it.

### Steps to reproduce

Click on dropdown.

### Did this work in previous version of GitExtensions?

Yes

**Comment by pmiossec** [comment, marked as answer]
You mean you didn't find the setting to change the order?

Search in the "Appearance" section a setting named "Sort branch by"

**Reply by RussKie** [reply]
![image](https://user-images.githubusercontent.com/4403806/138257524-174abb0c-6b56-4c79-9300-4abb1067d6e2.png)

**Reply by m0v3r** [reply]
Thanks!
